docs/index.html

```diff
@@ -51,765 +61,903 @@
           "link": ""
         },
         {
-          "label": "rust",
-          "ring": 2,
+          "label": "kotlin",
+          "ring": 0,
           "quadrant": 0,
           "active": "TRUE",
-          "moved": -1,
+          "moved": 0,
           "link": ""
         },
         {
-          "label": "kotlin",
+          "label": "typescript",
           "ring": 0,
           "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "typescript",
+          "label": "python",
           "ring": 0,
           "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "micronaut",
+          "label": "YARN",
           "ring": 0,
-          "quadrant": 1,
+          "quadrant": 0,
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
         {
-          "label": "angular",
+          "label": "NPM",
           "ring": 0,
-          "quadrant": 1,
+          "quadrant": 0,
           "active": "TRUE",
-          "moved": 0,
+          "moved": 1,
           "link": ""
         },
         {
-          "label": "SSR for small services",
+          "label": "NodeJS",
           "ring": 0,
-          "quadrant": 1,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
         },
         {
-          "label": "matomo",
+          "label": "Swift",
           "ring": 0,
-          "quadrant": 3,
+          "quadrant": 0,
           "active": "TRUE",
           "moved": 0,
           "link": ""
-        },
+        },        
         {
-          "label": "parquet",
+          "label": "Swift Package Manger",
```

Comment:
typo in manager, but also we should just use the actual name, as we do with others, eg NPM
```suggestion
          "label": "SwiftPM,
```

or alternatively

```suggestion
          "label": "Swift Package Manager,
```

Reply:
```suggestion
          "label": "Swift Package Manager",
```